docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -199,6 +200,78 @@ Default: `false`
 
 ---
 
+### Raft options
+
+{{< note title="Note" >}}
+
+Ensure that values used for Raft and the write ahead log (WAL) in `yb-tserver` configurations match the values in `yb-master` configurations.
```

Comment:
Oh damn, I did not realize we have two separate copies of the settings, for master and tserver. In practice, the good portion of the flags should apply to both master and tserver, as they cover knobs for the storage engine, which both require.